workflows/plugins/output_loaders/entities/consensus_genome_output.py

```diff
@@ -44,17 +44,28 @@ async def load(
             taxon_id = res["taxa"][0]["id"]
             accession_id = res["accessions"][0]["id"]
         else:
-            taxon_id = entity_inputs["taxon"].entity_id
-            accession_id = entity_inputs["accession"].entity_id
+            taxon_input = entity_inputs["taxon"]
+            assert isinstance(taxon_input, EntityInput)
+            taxon_id = taxon_input.entity_id
```

Comment:
would it be possible to name this taxon_entity_id or something? I think taxon_id might be confusing later. Same with accession_id.